example/lib/main.dart

```diff
@@ -4,11 +4,13 @@ import 'package:instabug_flutter/instabug_flutter.dart';
 
 Future<void> main() async {
   runApp(const MyApp());
-  Instabug.start(
-      'ed6f659591566da19b67857e1b9d40ab', [InvocationEvent.floatingButton]);
+  Instabug.init(
+    token: 'ed6f659591566da19b67857e1b9d40ab',
+    invocationEvents: [InvocationEvent.floatingButton],
+  );
   final client = InstabugHttpClient();
   final response = await client.get(Uri.parse('https://google.com'));
-  print(response.body);
+  debugPrint(response.body);
```

Comment:
I'd say let's move all analysis changes to a separate PR and make it the base for this one.

Reply:
To not delay this PR further, let's keep them here.